draft-ietf-gnap-core-protocol.md

```diff
@@ -2243,17 +2250,17 @@ code (string)
               minutes. It is RECOMMENDED that this code be no more than eight
               characters in length.
 
-url (string)
-: RECOMMENDED. The interaction URL that the client instance
-              will direct the RO to. This URL MUST be stable such
+uri (string)
+: RECOMMENDED. The interaction URI that the client instance
+              will direct the RO to. This URI MUST be stable such
```

Comment:
Revert parameter name change, will be addressed in with #123

Reply:
Reverted